src/commands/git/switch.ts

```diff
@@ -73,7 +73,7 @@ export class SwitchGitCommand extends QuickCommand<State> {
 			() =>
 				Promise.all(
 					state.repos.map(r =>
-						r.switch(state.reference.ref, { createBranch: state.createBranch, progress: false }),
+						r.switch(state.reference.ref, { createBranch: state.createBranch, progress: false, track: false }),
```

Comment:
Is this an intentional change in behavior? Because `switch` will now always add the `--no-track` flag, which I'm not sure is desired.